source/backend/dal/Repositories/AcquisitionFileRepository.cs

```diff
@@ -167,8 +167,12 @@ public LastUpdatedByModel GetLastUpdateBy(long id)
             lastUpdatedByAggregate.AddRange(ownersLastUpdatedBy);
 
             // Acquisition Deleted Owners
-            var ownersHistLastUpdatedBy = this.Context.PimsAcquisitionOwnerHists.AsNoTracking()
-                .Where(aoh => aoh.AcquisitionFileId == id)
+            var deletedOwners = this.Context.PimsAcquisitionOwnerHists.AsNoTracking()
+               .Where(aph => aph.AcquisitionFileId == id)
+               .GroupBy(aph => aph.AcquisitionOwnerId)
+               .Select(gaph => gaph.OrderByDescending(a => a.EffectiveDateHist).FirstOrDefault()).ToList();
```

Comment:
This pattern is used throught. groups by the historic table entry and selects the most recent for each of the group.

Reply:
I'll make sure to apply this to my PR.